apps/www/components/Article/Page.js

```diff
@@ -266,6 +266,7 @@ const ArticlePage = ({
   const articleContent = article?.content
   const articleUnreadNotifications = article?.unreadNotifications
   const routerQuery = router.query
```

Comment:
use !! since the isClimate prop on the frame expects a boolean (in case one of the ? returns undefined this will convert undefined -> false)